.github/workflows/tests.yml

```diff
@@ -21,9 +21,7 @@ jobs:
         with:
           python-version: "3.11"
       - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip uv
-          python -m uv pip install ruff==0.5.2
+        run: python -m pip install -r requirements/test.txt
       # Update output format to enable automatic inline annotations.
       - name: Run Ruff Linter
```

Comment:
instead of hard running it here, what do you think about setting up the https://pre-commit.com/ here, then after you guys set up the https://pre-commit.ci into the repo, so it enables the auto-fix feature on PR's...

Reply:
I have mixed feelings about this. I like the ability to auto-fix issues, just not auto-commits. I'm on the fence. We haven't enabled this in TorchGeo either.